Gruntfile.js

```diff
@@ -124,7 +132,7 @@ module.exports = function (grunt) {
       },
       jekyll: {
         files: ['source/**/*', '!source/_less/*', '!source/_site/**/*', '!source/assets/css/*.css',],
-        tasks: ['jekyll']
+        tasks: ['jekyll', 'copy:ngdocs']
```

Comment:
I think we could achieve this same result by using a symlink instead of a copy.  That's what I'm doing for the PatternFly dependencies.  See https://github.com/patternfly/patternfly-toolkit/blob/master/source/components